make.bat

```diff
@@ -397,4 +387,5 @@ if defined debug (ECHO Debug Mode = ON) else (ECHO Debug Mode = OFF)
 if defined debug (SET "debug=") else (SET "debug=%default%")
 goto menu
 
-:END
```

Comment:
This may cause unwanted termination of the script before the user is done with assembly. It is better to leave it the way it is.

Reply:
@Counterply, I didn't do anything to the end except put exit /b. I don't know why it is deleted.

Reply:
Is it ok, now?